### PR TITLE
Aorsa test

### PR DESCRIPTION
**Pull request by mbeidler3**
Merging old AORSA branches and developing test

### CODE REVIEW

**Comment by idigs** [comment]
Test this please

File: innocuous_file.txt
Comment (quantumsteve) [inline innocuous_file.txt:1]: remove?

**Comment by mbeidler3** [comment]
test this please

**Comment by mbeidler3** [comment]
test this please

**Comment by mbeidler3** [comment]
please test

**Comment by mbeidler3** [comment]
This is working on gnu 12.2 and nvhpc 23.9 (for cpu and gpu) on my machine and gnu 11.4 for CPU and nvhpc 24.5 for GPU on ExCL. This is only not working on github cloud (Agate) with gnu 13.2 and appears to be an issue with the openmp. I am going to merge this branch and the mars MPI branch and open a new pull request where I will gut the openmp now that it is not being used for CPU thread/vector parallelism. We will likely return to openmp for GPU offload in the future.

**Pull request merged**